backend/de.metas.cucumber/src/test/resources/de/metas/cucumber/features/createProductionOrder.feature

```diff
@@ -0,0 +1,423 @@
+      | oc_1       | p_1                     | bom_1                        | ppln_1                            | 540006        | 10         | 10           | 0            | PCE               | 2021-04-16T21:00:00Z | 2021-04-16T21:00:00Z | false     | false    |
+    And after not more than 10s, PP_OrderLine_Candidates are found
+      | PP_Order_Candidate_ID.Identifier | Identifier | M_Product_ID.Identifier | QtyEntered | C_UOM_ID.X12DE355 | ComponentType | PP_Product_BOMLine_ID.Identifier |
+      | oc_1                             | olc_1      | p_2                     | 100        | PCE               | CO            | boml_1                           |
+    And the following PP_Order_Candidates are enqueued
+      | PP_Order_Candidate_ID.Identifier |
+      | oc_1                             |
+    Then after not more than 10s, PP_Order_Candidates are found
+      | Identifier | M_Product_ID.Identifier | PP_Product_BOM_ID.Identifier | PP_Product_Planning_ID.Identifier | S_Resource_ID | QtyEntered | QtyToProcess | QtyProcessed | C_UOM_ID.X12DE355 | DatePromised         | DateStartSchedule    | Processed | IsClosed |
+      | ocP_1      | p_1                     | bom_1                        | ppln_1                            | 540006        | 10         | 0            | 10           | PCE               | 2021-04-16T21:00:00Z | 2021-04-16T21:00:00Z | true      | false    |
+    And after not more than 10s, PP_Orders are found
+      | Identifier | M_Product_ID.Identifier | PP_Product_BOM_ID.Identifier | PP_Product_Planning_ID.Identifier | S_Resource_ID | QtyEntered | QtyOrdered | C_UOM_ID.X12DE355 | C_BPartner_ID.Identifier | DatePromised         |
+      | ppo_1      | p_1                     | bom_1                        | ppln_1                            | 540006        | 10         | 10         | PCE               | endcustomer_2            | 2021-04-16T21:00:00Z |
+    And after not more than 10s, PP_Order_BomLines are found
+      | PP_Order_ID.Identifier | Identifier | M_Product_ID.Identifier | QtyEntered | IsQtyPercentage | C_UOM_ID.X12DE355 | ComponentType |
+      | ppo_1                  | ppol_1     | p_2                     | 100        | false           | PCE               | CO            |
+    And after not more than 10s, PP_OrderCandidate_PP_Order are found
+      | PP_Order_Candidate_ID.Identifier | PP_Order_ID.Identifier | QtyEntered | C_UOM_ID.X12DE355 |
+      | ocP_1                            | ppo_1                  | 10         | PCE               |
+    And after not more than 10s, MD_Candidates are found
+      | Identifier | Type   | BusinessCase | M_Product_ID.Identifier | Time                 | DisplayQty | ATP |
+      | c_3        | SUPPLY | PRODUCTION   | p_1                     | 2021-04-16T21:00:00Z | 10         | 0   |
+    And the following MD_Candidates are validated
```

Comment:
`"after not more than 30s, MD_Candidates are found"` searches for the latest MD_Candidate of a particular type and its related STOCK MD_Candidate ( no candidate Id involved )
 
`"the following MD_Candidates are validated"` gets the MD_Candidate of a particular type by id.

We need them both because depending on the step we are aware or not of the particular candidate id.

regarding column names:
  - on `"the following MD_Candidates are validated"` I've used the real db column names for the fields, so I'll update `"after not more than 30s, MD_Candidates are found"` in the same manner